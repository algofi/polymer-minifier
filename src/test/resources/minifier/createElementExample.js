<dom-module id="x-five-properties">
	<template>
		<script>
			Polymer( {
				is:'x-five-properties',
				ready: function () {
					var element = document.createElement( 'paper-checkbox');
					element.checked = true;
				}
			} );
		</script>
	</template>
</dom-module>
