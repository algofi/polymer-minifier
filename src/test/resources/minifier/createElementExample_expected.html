<dom-module id="x-five-properties">
	<template>
		<script>Polymer({is:"x-five-properties",ready:function(){var element=document.createElement("x-k");element.k=true;document.body.appendChild(element)}})</script>
	</template>
</dom-module>
